src/oci.h

```diff
@@ -374,6 +416,9 @@ struct cc_oci_config {
 	 */
 	struct cc_oci_vm_cfg           *vm;
 
+	/** Network configuration. */
+	struct cc_oci_net_cfg           net;
```

Comment:
This will end up being a pointer to multiple cc_oci_net_cfg when we support mutiple networks.